File: Java/src/Iniciante/Solucao_1009.java
import java.io.IOException;
import java.util.Scanner;

public class Solucao_1009 {

    public static double calcularSalarioComComisao(float salario, float vendas){
        final double comisao = 0.15;
        return salario + vendas * comisao;
    }

    public static void main(String[] args) throws IOException {
        var input = new Scanner(System.in);
        String nome;
        float salario,vendas;

        nome = input.next();
        salario = input.nextFloat();
        vendas = input.nextFloat();

        System.out.printf("TOTAL = R$ %.2f%n", calcularSalarioComComisao(salario,vendas));

    }

}

File: Python/Iniciante/Solucao_1009.py
def calcular_salario_com_comisao(salario, vendas):
    comisao = 0.15
    return  (salario + (vendas*comisao))
try:
    nome = input()
    salario = float(input())
    vendas = float(input())
    total = calcular_salario_com_comisao(salario,vendas)

    print(f"TOTAL = R$ {total:.2f}")
except (ValueError, TypeError):
    exit()